client/me/notification-settings/push-notification-settings/index.jsx

```diff
@@ -250,6 +250,7 @@ const PushNotificationSettings = React.createClass( {
 							) }</p>
 						</div>
 					</div>
+					<Notice className="notification-settings-push-notification-settings__instruction-refresh-notice" showDismiss={ false } text={ this.translate( 'Once you\'ve allowed notifications, you may need to refresh your browser.' ) } />
```

Comment:
This class should be showing an ESLint warning: `push-notification-settings` should be the prefix.